src/ypspur_ros.cpp

```diff
@@ -547,6 +553,9 @@ class YpspurRosNode
       subs_["cmd_vel"] = compat::subscribe(
           nh_, "cmd_vel",
           pnh_, "cmd_vel", 1, &YpspurRosNode::cbCmdVel, this);
+
+      pnh_.param("avoid_publishing_duplicated_odom", avoid_publishing_duplicated_odom_, false);
```

Comment:
This option may better to be enabled by default.

Reply:
Done